README.md

```diff
@@ -3,39 +3,34 @@
-
-**Please note that the tutorials presented here used version 0.2.0 of `icepyx`. These tutorials are set up to use that version in the provided Binder link, but they will not run without edits on more recent versions (>= v0.3.0) of `icepyx` due to non-backwards compatible changes in syntax. Up-to-date data access tutorials/examples are available [from `icepyx`](https://icepyx.readthedocs.io/en/latest/getting_started/example_link.html).**
+During the Hackweek participants worked in a [Pangeo](https://pangeo.io/) environment specifically created for the event and hosted on AWS us-west-2. The JupyterHub included a shared volume with datasets used during some tutorials that are too large to be included within this repository. However, these data are publicly available from [NSIDC](https://nsidc.org/data/icesat-2) and can be easily obtained using the [icepyx library](https://icepyx.readthedocs.io/en/latest/). Where possible, code to download the needed data using `icepyx` has been included within each tutorial notebook.
 
-To maximize the utility of this archive, most notebooks, including all those that require large datasets, have been rendered to include outputs. These notebooks can be identified by the "\_rendered" at the end of the filename.
+**Please note that the tutorials presented here used version 0.2.0 of `icepyx`. These tutorials are set up to use that version in the provided Binder link, but they will require modification for more recent versions (>= v0.3.0) of `icepyx`. Up-to-date data access tutorials/examples are available from the [`icepyx`](https://icepyx.readthedocs.io/en/latest/getting_started/example_link.html) repository and associated documentation.**
 
 ### Re-create the ICEsat-2 Hackweek JupyterLab environment with Binder
 Clicking this button will launch a [binder](https://mybinder.org/) replica of the [JupyterLab computing environment](https://github.com/ICESAT-2HackWeek/jupyterhub-2020) described above. With the exception of those tutorials denoted with an asterisk(\*), this will allow you to run the tutorials presented during the Hackweek. Be aware the session is ephemeral. **Your home directory will not persist, so use this binder only for running tutorials or other short-lived demos!**
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ICESAT-2HackWeek/2020_ICESat-2_Hackweek_Tutorials/binder?urlpath=git-pull%3Frepo%3Dhttps%253A%252F%252Fgithub.com%252FICESAT-2HackWeek%252F2020_ICESat-2_Hackweek_Tutorials%26urlpath%3Dlab%252Ftree%252F2020_ICESat-2_Hackweek_Tutorials%252F%26branch%3Dbinder)
 
-
 ## Tutorials
 \* Tutorial filenames within this repository denoted below with an asterisk(\*) cannot be run in Binder due to large dataset requirements.
 
 ### 01. Introductory Session (slides)
 *Anthony Arendt and Charley Haley*
-
```

Comment:
If I recall correctly, I initially didn't have these blank lines here and it interfered with the rendering of the bullets...

Reply:
Ah, OK, yeah, I think there are potentially issues around single-item lists.  I think adding two spaces to end of the preceding line might work.  Maybe someday I'll get past the trial and error phase of markdown nuances like this :)